### DISCUSSION TITLE
Sudo as OP

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#3732] [Feature Requests]
We can make player before commands

/cmi sudo [playerName] (command/c:[text]) - forces player to perform defined command or to send public message

What would be needed is an axtra variable like **_as:op_**  that would allow us to let the player execute a command he usully as no permission for. Its important that the player execute the command itself therefor we use the sudo.

_/cmi sudo [playerName] **(as:op)** (command/c:[text]_
as:op - execute as when he is OP with all rights

silly example:
/cmi sudo Cloude_Lecaw as:op ban Cloude_Lecaw i want to ban myself.

**Comment by mrfloris** [comment]
Personally I think when you get to this point the approach of what you wish to achieve for a feature needs to change. For example have a command that uses asConsole! etc.
